assets/javascript/app.js

```diff
@@ -1,4 +1,43 @@
 //= require_directory ./vendor
 //
 
-// custom JS can go in this file, or another, then you add another require line
\ No newline at end of file
+// custom JS can go in this file, or another, then you add another require line
+
+function showCareerPrev(currIndex) {
+  'use strict';
+
+  var newIndex = parseInt(currIndex) - 1;
+  if (newIndex > 0) {
+    toggleCareer(currIndex, newIndex);
+  }
+}
+
+function showCareerNext(currIndex, count) {
```

Comment:
The prefix isn't really important here for any particular reason. Maybe if the codebase was much larger..